stylelint-config.json

```diff
@@ -28,7 +28,9 @@
 
     "property-case": "lower",
 
-    "declaration-block-no-duplicate-properties": true,
+    "declaration-block-no-duplicate-properties": [ true, {
+      "ignore": ["consecutive-duplicates"]
```

Comment:
So I actually tried that and I got: Invalid value "consecutive-duplicates-with-different-values", maybe we're not using the latest version of stylelint?